### ISSUE TITLE
Duplicate Results?

### ISSUE THREAD

**Issue by cbizon** [ranking-agent/strider#125]
This query seems to be generating some duplicate results:
```
{
    "message": {
        "query_graph": {
            "nodes": {
                "a": {
                    "category": "biolink:Disease"
                },
                "b": {
                    "category": "biolink:Disease",
                    "id": "MONDO:0019052"
                }
            },
            "edges": {
                "ab": {
                    "subject": "a",
                    "object": "b",
                    "predicate": "biolink:subclass_of"
                }
            }
        }
    },
    "knowledge_graph": {
        "nodes": [],
        "edges": []
    },
    "results": []
}
```

In particular I see Fanconi Anemia (same ID) appearing multiple times in the results (at least 3).  Two of them seem to be different sources, BTE and SRI KG.  But it also looks like SRI KG has two edges?  I'm looking at the results in the RTX viewer [here](https://arax.ncats.io/?source=ARS&id=9381dd4b-4f26-42fb-b3e0-9bd4997acd51), so there may be some other edge properties preventing a merge that I am missing.

**Comment by alongreyber** [comment]
I believe this was solved by #119. I ran this query and I can see only one result with a binding to MONDO:0013566 (Fanconi Anemia). I'm not 100% sure so I'll leave this open until confirmed by @cbizon.

[Output from the latest master branch](https://github.com/ranking-agent/strider/files/6286864/response_1617981781881.json.zip)

**Comment by patrickkwang** [comment]
Looks good to me.
